deps/graph-parser/src/logseq/graph_parser/block.cljs

```diff
@@ -181,6 +181,20 @@
          (remove string/blank?)
          distinct)))
 
+(defn- extract-block-refs
+  [nodes]
+  (let [ref-blocks (atom nil)]
+    (walk/postwalk
```

Comment:
For perf reasons I wish we weren't using postwalk but without a good understanding of every ast node combination, this is what we have to do

Reply:
I plan to add the ast malli schema later. 
we can leave this 'walk' for now